ocaml/xapi-guard/lib/disk_cache.ml

```diff
@@ -0,0 +1,341 @@
+let ( // ) = Filename.concat
+
+let runtime_data = "/var/lib" // "xapi-guard"
+
+let ( let* ) = Lwt.bind
+
+let ( let@ ) f x = f x
+
+let with_lock = Lwt_mutex.with_lock
+
+type t = Uuidm.t * Mtime.t
```

Comment:
`t` is the type of elements in the queue rather than the type of the cache itself or elements in the cache?

What exactly is the uuid from?

Reply:
The uuid is the VM uuid. `t` is the type of the requests / elements in the cache.

Reply:
...but does not include the data of the elements in the cache.